### PR TITLE
Translation agent updates and add Maps agent

### PR DESCRIPTION
**Pull request by mihirapatel**
Updated translation agent and added initial code for maps agent
Updated tip, unit, currency agents

### CODE REVIEW

**Comment by mihirapatel** [comment]
@googlebot I fixed it.

**Comment by mihirapatel** [comment]
@googlebot I fixed it.

File: portfolio/src/main/java/com/google/sps/agents/Language.java [hunk 1]
```diff
@@ -21,21 +21,21 @@
   	private String searchText;
```
Comment (abbymapes) [inline portfolio/src/main/java/com/google/sps/agents/Language.java:21]: Don't need this?

File: portfolio/src/main/java/com/google/sps/agents/Maps.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import java.io.IOException;
+import java.util.Map;
+ 
+/**
+ * Maps Agent
+ */
+public class Maps implements Agent {
+    
+    private final String intentName;
+    private String searchString;
```
Comment (abbymapes) [inline portfolio/src/main/java/com/google/sps/agents/Maps.java:19]: Don't need this?

File: portfolio/src/main/java/com/google/sps/agents/Name.java [hunk 3]
```diff
@@ -18,8 +18,8 @@
   	private String searchText;
```
Comment (abbymapes) [inline portfolio/src/main/java/com/google/sps/agents/Name.java:18]: don't need this

File: portfolio/src/main/java/com/google/sps/agents/Reminders.java [hunk 4]
```diff
@@ -18,8 +18,8 @@
   	private String searchText;
```
Comment (abbymapes) [inline portfolio/src/main/java/com/google/sps/agents/Reminders.java:18]: Don't need this?

File: portfolio/src/main/java/com/google/sps/agents/Currency.java [hunk 5]
```diff
@@ -20,20 +20,20 @@
...
 
     public Currency(String intentName, Map<String, Value> parameters) {
-      this.intentName = intentName;
-      setParameters(parameters);
+        this.intentName = intentName;
+        setParameters(parameters);
     }
 
 	@Override 
 	public void setParameters(Map<String, Value> parameters) {
```
Comment (aliu917) [inline portfolio/src/main/java/com/google/sps/agents/Currency.java:28]: indentation?

**Pull request merged**